psa-crypto-sys/build.rs

```diff
@@ -357,6 +361,11 @@ mod operations {
             let mut mbed_lib_dir = compile_mbed_crypto()?;
             let mut mbed_include_dir = mbed_lib_dir.clone();
             mbed_lib_dir.push("lib");
+            if !mbed_lib_dir.as_path().exists() {
+                _ = mbed_lib_dir.pop();
+                mbed_lib_dir.push("lib64");
```

Comment:
On my machine, it is just "lib". Is this a special case?

Reply:
> On my machine, it is just "lib". Is this a special case?

Yeah when I updated mbedtls on my Fedora 39 machine, it uses lib64 over lib.